includes/Modules/Sign_In_With_Google/Settings.php

```diff
@@ -28,10 +28,23 @@ class Settings extends Module_Settings {
 	const TEXT_SIGN_IN_WITH_GOOGLE  = 'Sign in with Google';
 	const TEXT_SIGN_UP_WITH_GOOGLE  = 'Sign up with Google';
 
+	const TEXT_VALUES_MAP = array(
```

Comment:
I noticed that the options that [the SiwG button was expecting](https://developers.google.com/identity/gsi/web/reference/html-reference#data-theme) are different to the labels in SK, so mapping here.

Reply:
@benbowler, let's fix it correctly then. We need revert your changes in this file, update incorrect values in this file to use proper ones, and update js files to have user friendly labels and proper values.